source/tests/system/nifgen_driver_api_tests.cpp

```diff
@@ -80,7 +85,7 @@ class nifgenDriverApiTest : public ::testing::Test {
     ::grpc::Status status = GetStub()->InitiateGeneration(&context, request, &response);
 
     EXPECT_TRUE(status.ok());
-    EXPECT_EQ(kfgenDriverApiSuccess, response.status());
+    expect_api_success(response.status());
```

Comment:
This is for my understanding. Why did we replace   `EXPECT_EQ(kfgenDriverApiSuccess, response.status());` with `expect_api_success(response.status());` ?

Reply:
I changed that to be consistent with other driver tests. others are using that function